db/migrate/20150906173134_create_teams.rb

```diff
@@ -0,0 +1,13 @@
+class CreateTeams < ActiveRecord::Migration
+  def change
+    create_table :teams do |t|
+      t.belongs_to :account
+      t.belongs_to :team
```

Comment:
Look at the second-to-last commit - the way I defined the `account` and `team` associations lets us access parent/sub-teams with `team.parent_team` and `team.subteams`.